### PR TITLE
Feat/data seeder

### PR DESCRIPTION
**Pull request by DeannaShort**
1) Create Data Seeder class 
2) Create dataContext extension method to seed Users and Roles (currently Dev User and two roles: Basic and Admin)
3) Implement data seeder with SeedData method on Program.cs
4) Give every created user the role of Basic

### CODE REVIEW

**Review by xalluna — APPROVED** [review]

**Review by zoe-klein — APPROVED** [review]
😎

File: TcgPocket/Data/DataSeeder.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        };
+
+        var devAccount = await userManager.FindByEmailAsync(devAccountUser.Email);
+
+        if(devAccount is null)
+        {
+            await userManager.CreateAsync(devAccountUser, "Password1!");
+            userManager.AddToRoleAsync(devAccountUser, "Admin");
+        }
+    }
```
Comment (gbrad777) [inline TcgPocket/Data/DataSeeder.cs:43]: slightly concerned about this being in our repo for if we ever end up deploying the app, im guessing we would remove this or set it to only go if its in a "DEV" environment or something like that if the need arises?

Comment (DeannaShort) [reply TcgPocket/Data/DataSeeder.cs:43]: Yeah basically

Comment (gbrad777) [reply TcgPocket/Data/DataSeeder.cs:43]: otay, then im cool 👌

**Pull request merged**